.github/workflows/release.yml

```diff
@@ -109,17 +109,18 @@ jobs:
           GH_TOKEN: ${{ github.token }}
           VERSION: ${{ env.NPM_VERSION }}
         run: |
+
                echo "COMMIT_MESSAGE=$COMMIT_MESSAGE"
                git config --global user.name "github-actions[bot]"
                git config --global user.email "github-actions[bot]@users.noreply.github.com"
-               git tag -a "go/${VERSION}" -m "${COMMIT_MESSAGE}"
-               git tag -a "${VERSION}" -m "${COMMIT_MESSAGE}"
                git commit -m "[Automated changes] ${COMMIT_MESSAGE}"
-               git push origin --tags HEAD:master
+               git push origin HEAD:master
+               git tag -a "${VERSION}" -m "${COMMIT_MESSAGE}"
+               git push --tags
                echo "Creating github release..."
                gh release create "${VERSION}" --generate-notes --verify-tag
-               git push
-
+               git tag -a "go/${VERSION}" -m "${COMMIT_MESSAGE}"
```

Comment:
@pcriadoperez the go/X tag should be created first so it does not show up as the latest tag in the repo no?

Reply:
@pcriadoperez 

> Can we delete the go tag? 

I don't think so, without it pkg does not recognize the versioning

Reply:
but we can probably keep this order (version and then go/version) as long as the release uses `version` only